omod/src/main/webapp/pages/metadata/locations/locationAttributeType.gsp

```diff
@@ -100,7 +100,7 @@
             formFieldName: "name",
             id           : "name",
             maxLength    : 101,
-            initialValue : ui.encodeHtmlContent((locationAttributeType.name ?: ''))
```

Comment:
Could you throw some more light as to why here you are replacing encodeHtmlContent with encodeHtmlAttribute and yet further down, you are doing the reverse?